en/advanced-topics/jsb-web-difference.md

```diff
@@ -0,0 +1,88 @@
+# The differences between Pure JS and JS-Binding objects
+
+## Properties
```

Comment:
It is better to describe the main purpose of the documentation, for example

```
Because xxx, if you want your program run on all platforms(especially on native platforms), should not use some yyy features.
```